trackthenews/core.py

```diff
@@ -487,15 +469,17 @@ def initial_setup():
     else:
         try:
             os.makedirs(home)
-        except:
+        except Exception:
```

Comment:
This satisfies flake8's very reasonable check to not use bare exceptions (which get in the way of program interrupts and such).